### ISSUE TITLE
Exception Thrown in Example2 ComboBox in Windows 64 bit / Visual Studio 2015 Ver 1.2.4  + current master

### ISSUE THREAD

**Issue by JonMoore75** [actsl/kiss_sdl#14]
Tried to run example 2 in Visual Studio 2015 Community.

Found if you select anything but the first entry (eg select Baltimore) of the combo box the app crashes with exception thrown by combobox_event function.  Appears the bsearch is returning a pointer to unreadable (maybe unallocated memory), haven't managed to diagnose this further.

PS Also noticed that Linker complains of no entry point, easily fixed by setting Linker > System > Subsystem to WINDOWS, not sure if that is intended?

**Comment by JonMoore75** [comment]
Ok my mistake hadn't noticed in the manual till now that this must be run only in x86.  That fixes the issue.

So I'll close this but would be great to have it work in x64 too

**Comment by actsl** [comment]
It runs in x64, i run it in Linux and my Linux is 64 bit, and it runs there without issues.

The only difference between x86 and x64 in MinGW that i have is -m32 and -march=i686 on the command line, there are no these switches for 64 bit. In Visual Studio these should be set just by creating a project for x64, but in Visual Studio it is more complicated, as there are also other linked libraries for every Windows project, that are also different for x86 and x64.

Make a project for x64, then add the SDL paths the same as in x86 (except the library paths for x64), it **must** work for 64 bit.

**Comment by actsl** [comment]
I now made the new version of kiss_sdl 1.2.6. But the only change i made was changing the Visual Studio project files for SDL 2.0.5.

In spite that the Visual Studio projects created for x86 also contain settings for x64, these settings are a copy of x86 settings and are not correct for x64. I added the correct SDL 2.0.5 paths to the x64 settings, but all x64 settings are still not correct and thus building for x64 with these Visual Studio project files may cause all kind of issues.

As the author of this issue correctly said, in Visual Studio this project was made only for x86 and it is nowhere said that it should build for x64 with the provided project files. For building for x64, make a project in Visual Studio for x64, then add the correct SDL paths, then it must work for x64.

I now close this issue, as this is not an issue in the project the way it was made.

It would be great though if someone who has a 64 bit version of Windows, could provide Visual Studio project files for x64, they may then be added to the project. Many thanks for writing about the problems you had though, as this enabled to clarify these things to other people, as well as many thanks in advance to everyone who would report any problems one could have after doing as it was recommended here. No bugs have been found in kiss_sdl for more than a year, and together we can make sure that there will be none.
